### ISSUE TITLE
Update to the latest Sparkle version

### ISSUE THREAD

**Issue by Kavakuo** [JohnCoates/Aerial#763]
The newest Sparkle release is 1.21.3, according to the `Podfile.lock` file, Aerial uses Sparkle v1.21.0. Security critical frameworks, in this case for automatic updates, should always be up to date or at least updated regularly. There were already security problems with Sparkle [in the past](https://vulnsec.com/2016/osx-apps-vulnerabilities/).

Maybe this also helps to resolve issues like #759.

**Comment by glouel** [comment]
Yes, there was no security related bug so I kept the same version throughout the beta process, but of course I'll keep it updated going forward. I've issued a 1.4.10beta1 with Sparkle 1.21.3 here : https://github.com/JohnCoates/Aerial/releases/tag/v1.4.10beta1

Regarding the Chrome uncommon issue, since the betas weren't flagged (or unflagged as they got more downloads ?) and all used the same Sparkle version, including beta 12 from 2 days ago, I doubt it will change much. A quick check show no change here.
